bot/resources/evergreen/trivia_quiz.json

```diff
@@ -604,5 +604,309 @@
       "question": "When DNA is unzipped, two strands are formed. What are they called (separate both answers by the word \"and\")?",
       "answer": "leading and lagging, leading strand and lagging strand"
     }
+  ],
+  "cs": [
+    {
+      "id": 401,
+      "question": "What does HTML stand for?",
+      "answer": "Hyper Text Markup Language"
+    },
+    {
+      "id": 402,
+      "question": "What does ASCII stand for?",
+      "answer": "American Standard Code for Information Interchange"
+    },
+    {
+      "id": 403,
+      "question": "What does SASS stand for?",
+      "answer": "Syntactically Awesome Stylesheets, Syntactically Awesome Style Sheets"
+    },
+    {
+      "id": 404,
+      "question": "In neural networks, _______________ is an algorithm for supervised learning using gradient descent.",
+      "answer": "backpropagation"
+    },
+    {
+      "id": 405,
+      "question": "What is computing capable of performing exaFLOPS called?",
+      "answer": "exascale computing, exascale"
+    },
+    {
+      "id": 406,
+      "question": "In quantum computing, what is the full name of \"qubit\"?",
+      "answer": "quantum binary digit"
+    },
+    {
+      "id": 407,
+      "question": "Given that January 1, 1970 is the starting epoch of time_t in c time, and that time_t is stored as a signed 32-bit integer, when will unix time roll over (year)?",
+      "answer": "2038"
+    },
+    {
+      "id": 408,
+      "question": "What are the components of digital devices that make up logic gates called?",
+      "answer": "transistors"
+    },
+    {
+      "id": 409,
+      "question": "How many possible public IPv6 addresses are there (answer in 2^n)?",
+      "answer": "2^128"
+    },
+    {
+      "id": 410,
+      "question": "A hypothetical point in time at which technological growth becomes uncontrollable and irreversible, resulting in unforeseeable changes to human civilization is termed as...?",
+      "answer": "technological singularity, singularity"
+    },
+    {
+      "id": 411,
+      "question": "In cryptography, the practice of establishing a shared secret between two parties using public keys and private keys is called...?",
+      "answer": "key exchange"
+    },
+    {
+      "id": 412,
+      "question": "How many bits are in a TCP checksum header?",
+      "answer": "16, sixteen"
+    },
+    {
+      "id": 413,
+      "question": "Which protocol handles communication between email servers?",
+      "answer": "SMTP"
```

Comment:
This could be changed to "Give one protocol that handles mail" as there are SMTP, IMAP and POP3 that are used today